.travis.yml

```diff
@@ -7,15 +7,22 @@ matrix:
       services:
         - docker
       env: DOCKER_IMAGE=quay.io/pypa/manylinux1_x86_64
+           PLAT=manylinux1
     - sudo: required
       services:
         - docker
       env: DOCKER_IMAGE=quay.io/pypa/manylinux1_i686
            PRE_CMD=linux32
+           PLAT=manylinux1
+    - sudo: required
+      services:
+        - docker
+      env: DOCKER_IMAGE=quay.io/pypa/manylinux2010_x86_64
+           PLAT=manylinux1
```

Comment:
2010?

Also, the options for auditwheel need the full platform tag, like `manylinux2010_x86_64`.

Reply:
fixing. What is the 32 bit manylinux1_* full tag?

Reply:
trying the docker image name